scripts/curate_basel.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import json
+import argparse
+
+root_data = '/Users/alex/data/basel-mp2rage_raw'
+output_data = '/Users/alex/data/basel-mp2rage_raw_curated'
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(root_data, output_data):
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data)
+
+    dict_images = {
+        "MP2RAGE_UNI_Images.nii.gz": "_UNIT1.nii.gz"
+    }
+
+    dict_der = {
+        "lesion_mask_sc.nii.gz": "_UNIT1_lesion-manual.nii.gz"
+    }
+
+    for dirs, subdirs, files in os.walk(root_data):
+        for file in files:
+            if file.endswith('.nii.gz') and file in dict_images.keys() or file in dict_der.keys():
+                path_file_in = os.path.join(dirs, file)
+                path = os.path.normpath(path_file_in)
+                subid_bids = 'sub-' + (path.split(os.sep))[5]
+                flag_der = False
+                if file.endswith('lesion_mask_sc.nii.gz'):
+                    path_subid_bids_dir_out = os.path.join(output_data, 'derivatives', 'labels', subid_bids, 'anat')
+                    flag_der = True
+                else:
+                    path_subid_bids_dir_out = os.path.join(output_data, subid_bids, 'anat')
+                if not os.path.isdir(path_subid_bids_dir_out):
+                    os.makedirs(path_subid_bids_dir_out)
+                if flag_der is False:
+                    path_file_out = os.path.join(path_subid_bids_dir_out, subid_bids + dict_images[file])
+                else:
+                    path_file_out = os.path.join(path_subid_bids_dir_out, subid_bids + dict_der[file])
```

Comment:
Also you don't need `flag_der`. It's emulating control flow, but the flow it is controlling is simple enough to inline:

```suggestion
                if file.endswith('lesion_mask_sc.nii.gz'):
                    path_subid_bids_dir_out = os.path.join(output_data, 'derivatives', 'labels', subid_bids, 'anat')
                    path_file_out = os.path.join(path_subid_bids_dir_out, subid_bids + dict_der[file])
                else:
                    path_subid_bids_dir_out = os.path.join(output_data, subid_bids, 'anat')
                    path_file_out = os.path.join(path_subid_bids_dir_out, subid_bids + dict_images[file])
                if not os.path.isdir(path_subid_bids_dir_out):
                    os.makedirs(path_subid_bids_dir_out)
```

Reply:
Fixed here https://github.com/neuropoly/data-management/pull/91/commits/ab32323f5d71794c68c6b6522906019a7eb475d6